src/ar/mangalek/src/eu/kanade/tachiyomi/extension/ar/mangalek/Mangalek.kt

```diff
@@ -21,11 +21,21 @@ import uy.kohesive.injekt.api.get
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-private const val mangalekUrl = "https://lekmanga.net"
+private const val MIRROR_PREF_KEY = "MIRROR"
+private const val MIRROR_PREF_TITLE = "تعديل رابط مانجا ليك"
+internal val MIRROR_PREF_ENTRY_VALUES = arrayOf(
+    "https://lekmanga.net",
+    "https://lekmanga.org",
+    "https://like-manga.net",
+    "https://lekmanga.com",
+)
+private val MIRROR_PREF_DEFAULT_VALUE = MIRROR_PREF_ENTRY_VALUES[0]
+private const val RESTART_TACHIYOMI = ".لتطبيق الإعدادات الجديدة Tachiyomi أعد تشغيل"
+
 class Mangalek :
     Madara(
         "مانجا ليك",
-        mangalekUrl,
+        MIRROR_PREF_DEFAULT_VALUE,
         "ar",
         SimpleDateFormat("MMMM dd, yyyy", Locale("ar")),
```

Comment:
Doesn't work for `lekmanga.org`, which uses `yyyy-MM-dd`, not sure if there's a decent approach to handling that.

Reply:
can override `parseChapterDate` from Madara and try both formats; if first results in 0, try second one

Reply:
I have no idea how to fix it, i tried many solutions but that's not work.

Reply:
```
private val formatOne = SimpleDateFormat("MMMM dd, yyyy", Locale.US)
private val formatTwo = SimpleDateFormat("yyyy-MM-dd", Locale.US)

override fun parseChapterDate(date: String?): Long {
    date ?: return 0L

    return try {
        formatOne.parse(date)!!.time
    } catch (_: ParseException) {
        try {
        	formatTwo.parse(date)!!.time
        } catch (_: ParseException) {
            0L
        }
    }
}
```
try this

Reply:
> ```
> private val formatOne = SimpleDateFormat("MMMM dd, yyyy", Locale("ar"))
> private val formatTwo = SimpleDateFormat("yyyy-MM-dd", Locale.US)
> 
> override fun parseChapterDate(date: String?): Long {
>     date ?: return 0L
> 
>     return try {
>         formatOne.parse(date)!!.time
>     } catch (_: ParseException) {
>         try {
>         	formatTwo.parse(date)!!.time
>         } catch (_: ParseException) {
>             0L
>         }
>     }
> }
> ```
> 
> try this

ok, thank you